tests/sentry/incidents/serializers/test_workflow_engine_base.py

```diff
@@ -25,6 +25,10 @@
 class TestWorkflowEngineSerializer(TestCase):
     @assume_test_silo_mode(SiloMode.REGION)
     def setUp(self) -> None:
+        # XXX: do this so that DCGA and Action IDs aren't one to one
+        other_action = self.create_action()
+        other_action.delete()
```

Comment:
You could also manually update the id
https://github.com/getsentry/sentry/blob/68459a2224e0826b438de3da76fe8c10b61632a7/tests/sentry/users/api/endpoints/test_user_authenticator_enroll.py#L531-L535